packages/grafana-ui/src/components/Forms/InlineLabel.story.tsx

```diff
@@ -12,18 +12,54 @@ const meta: ComponentMeta<typeof InlineLabel> = {
       page: mdx,
     },
   },
+  argTypes: {
+    children: {
+      control: 'text',
+    },
+    tooltip: {
+      control: 'text',
+    },
+    width: {
+      control: 'text',
+    },
+    transparent: {
+      control: 'boolean',
+    },
+    interactive: {
+      control: 'boolean',
+    },
+    as: {
+      table: {
+        disable: true,
+      },
+    },
```

Comment:
An easy way to do this is is adding this to the same level as `argTypes`
```
    controls: {
      exclude: ['as'],
    },
```

Reply:
This is awesome, for some reason, I didn't come across it when looking through the storybook docs! Thanks!